package.json

```diff
@@ -34,6 +34,7 @@
     "format-json": "^1.0.3",
     "identity-obj-proxy": "^3.0.0",
     "lodash": "^4.17.11",
+    "lodash.debounce": "^4.0.8",
```

Comment:
I got told that this was missing so I installed it just to get the app launching